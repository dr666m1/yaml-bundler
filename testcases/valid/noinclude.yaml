foo:
  bar: &bar baz
qux: *bar
