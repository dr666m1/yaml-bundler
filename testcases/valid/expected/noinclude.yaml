foo:
  bar: baz
qux: baz
